### ISSUE TITLE
Doc for adding events is off, or there is some other bug

### ISSUE THREAD

**Issue by acim1** [reagent-project/reagent-forms#151]
https://github.com/reagent-project/reagent-forms#adding-events

For the below component, the "id" parameter supplied in the event function was actually a sequence (or `(:day)`, when changing the day). That's why I changed the name to `ids` to make it less confusing for myself, and updated the logic accordingly. The given is what worked to fire the event when changing day:

```
(defn day-selector-component []
  [:div
   [:label "Select a day"]
   [bind-fields
    [:select {:field :list :id :day}
     (for [opt (range 1 26)] [:option {:key (keyword (str opt))} opt])]
    state
    (fn [ids _ doc]
      (when (some #{:day} ids)
        (assoc-in doc [:result :status] nil)))]])
```

**Comment by yogthos** [comment]
The reason is that the `:id` represents a path into the document. If you change your select `:id` key to be `:calendar.id`, then you'll see `(:calendar :id)` as the first argument.

**Comment by yogthos** [comment]
Just released `0.5.43` with the new API for events. The new version event handler signature accepts the following arguments:

```clojure
(fn [id path value doc]
      (println id doc)
      (case id
        :calendar.day
        (assoc-in doc [:result :status] :hello)
        value))
```

The `path` contains the path for looking up the field in the document, and the `id` contains the `:id` of the field in the form.
